deploy.sh

```diff
@@ -150,7 +150,18 @@ fi
 svn status
 
 echo "➤ Committing files..."
-svn commit -m "Update to version $VERSION from GitHub" --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+if [[ "$INPUT_COMMIT_MESSAGE" != false ]]; then
+	if [[ "$INPUT_COMMIT_MESSAGE" = true ]]; then
+		#.git DIR still in place, so we should be able to retrieve the last message
+		MESSAGE=$(git -C "$GITHUB_WORKSPACE" log -1 --format=%s)
+	else
+		#provided commit message
+		MESSAGE=$INPUT_COMMIT_MESSAGE
```

Comment:
Look like you want to support the case where user can provide their own commit message here. I'm sorry I didn't mention this in the previous review.

If users use this flag and pass an actual message, the message will be the same for all deployments. IMO, it's not very helpful, so I think we should remove these lines too:

> Use the latest git commit message as the svn commit message, otherwise use "Update to version $VERSION from GitHub". Defaults to false.

Reply:
Please refer to the example i provided [here](https://github.com/10up/action-wordpress-plugin-deploy/pull/94#discussion_r888033202) for how I am setting the version. It is possible to set a dynamic commit message either using output from another action or setting an environment variable before running this action. If you check the example you would be able to better understand.

[This](https://github.com/richard-muvirimi/Woo-Custom-Gateway/blob/production/.github/workflows/deploy.yml) is an action I have setup which actually resolves the version number on the fly.